app/validators/list.ts

```diff
@@ -37,8 +37,7 @@ export default function validateList(validator: ValidatorFunction): ListValidato
  * results that are not arrays (and thus aren't for individual list items).
  */
 export function transposeResults(results?: Array<ValidatorResult | ValidatorResult[]>) {
-    const transposer = (acc: ValidatorResult[][], itemErrors: ValidatorResult[]) => (
-        Array.isArray(itemErrors) ? itemErrors.map((val, index) => [...(acc[index] || []), val]) : acc
-    );
+    const transposer = (acc: ValidatorResult[][], itemErrors: ValidatorResult[]) =>
+        itemErrors.filter(Array.isArray).map((val, index) => [...(acc[index] || []), val]);
```

Comment:
Wait this commit changed the logic -- it used to be that `itemErrors` may-or-may-not-be an array, but now it's `itemErrors`' items that may-or-may-not-be arrays. I have no idea which is correct.

Reply:
doh! Fixed in latest commit.